Installation/Source/xxllnc.zaaksysteemv2.source.json

```diff
@@ -5,7 +5,8 @@
     "version": "0.0.1",
     "description": "xxllnc zaaksysteem v2 api",
     "location": "!ChangMe!",
```

Comment:
Reminder: for any test or acc source it is oke and often helpfull to add a location, just don't add any auth